### PR TITLE
feat: 5384 bridge test to x1 testnet via fast path

### PR DESCRIPTION
**Pull request by liu-zhipeng**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/deployments/contracts/src/cli/init/init.ts [hunk 1]
```diff
@@ -454,6 +454,7 @@ export const initProtocol = async (protocol: ProtocolStack, apply: boolean, stag
           // Allowlist watchers in WatcherManager + Connext.
           for (const watcher of protocol.agents.watchers.allowlist) {
             for (const network of protocol.networks) {
+              if (network.chain !== "195") continue;
```
Comment (LayneHaber) [inline packages/deployments/contracts/src/cli/init/init.ts:457]: shouldn't this be
```suggestion
              if (network.chain === "195") continue;
```

or deleted if we don't want to special case? (we don't really need to leave this in here, as we will be shutting down this testnet-prod env)

File: packages/deployments/subgraph/config/amarok-runtime-v0.json [hunk 2]
```diff
@@ -250,5 +250,26 @@
         "startBlock": 13383551
       }
     ]
+  },
+  {
+    "subgraphName": "connext/amarok-runtime-v0-x1-testnet",
+    "network": "xgon",
```
Comment (LayneHaber) [inline packages/deployments/subgraph/config/amarok-runtime-v0.json:256]: is this the right network name?

File: packages/deployments/subgraph/src/amarok-hub-v0/subgraph.template.yaml [hunk 3]
```diff
@@ -233,3 +233,57 @@ dataSources:
...
+        - Transfer
+      abis:
+        - name: BaseHubConnector
+          file: ./src/amarok-hub-v0/abis/OptimismHubConnector.json
+      eventHandlers:
+        - event: MessageProcessed(bytes,address)
+          handler: handleBaseMessageProcessed
+        - event: NewConnector(indexed uint32,indexed uint32,address,address,address)
+          handler: handleBaseNewConnector
+      file: ./src/amarok-hub-v0/mapping/base.ts
```
Comment (LayneHaber) [inline packages/deployments/subgraph/src/amarok-hub-v0/subgraph.template.yaml:262]: should this be included in this PR?

File: packages/deployments/subgraph/config/amarok-runtime-v0.json (hunk 2 above)
Comment (liu-zhipeng) [reply packages/deployments/subgraph/config/amarok-runtime-v0.json:256]: Yes

File: packages/deployments/contracts/src/cli/init/init.ts (hunk 1 above)
Comment (liu-zhipeng) [reply packages/deployments/contracts/src/cli/init/init.ts:457]: deleted!

File: packages/deployments/subgraph/src/amarok-hub-v0/subgraph.template.yaml (hunk 3 above)
Comment (liu-zhipeng) [reply packages/deployments/subgraph/src/amarok-hub-v0/subgraph.template.yaml]: deleted

**Review by wanglonghong — APPROVED** [review]

**Review by LayneHaber — APPROVED** [review]

**Review by preethamr — APPROVED** [review]

**Pull request merged**